migrations/011-user-result-highlights.js

```diff
@@ -0,0 +1,17 @@
+module.exports.up = function (db) {
+  db.prepare(
+    `
+    create table "UserResultHighlight" (
+      "teamId" integer not null,
+      "userId" integer not null,
+      foreign key ("teamId") references "CalendarEventResultTeam"("id") on delete cascade,
+      foreign key ("userId") references "User"("id") on delete cascade,
```

Comment:
Should add index on these FK's. On some migrations I forgot it... `007-builds.js` is an example of one that has them.

Reply:
Will add them! I'm not that well-versed in SQL so I'm flailing a lot and am usually already happy if things work :D